c2corg_ui/static/js/outingediting.js

```diff
@@ -147,10 +147,6 @@ app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
       outing.date_end = window.moment(outing.date_end).format('YYYY-MM-DD');
     }
 
-    var associations = outing.associations;
-    for (var i = 0; i < associations['users'].length; i++) {
-      associations['users'][i]['id'] = associations['users'][i]['document_id'];
```

Comment:
You removed a part of the code but added nothing back - is this one already done elsewhere?